src/App.tsx

```diff
@@ -20,12 +32,13 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={event => setQuery(event.target.value)}
```

Comment:
[CODE STYLE] - Avoid using one-letter variable naming. It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search. Consider using a more descriptive name such as `event` instead of `e`.